maro/cli/data_pipeline/vm_scheduling.py

```diff
@@ -0,0 +1,296 @@
+import gzip
+import os
+import shutil
+import time
+from csv import reader, writer
+from typing import List
+
+import aria2p
+import pandas as pd
+from yaml import safe_load
+
+from maro.cli.data_pipeline.base import DataPipeline, DataTopology
+from maro.cli.data_pipeline.utils import StaticParameter, convert, download_file
+from maro.utils.exception.cli_exception import CommandError
+from maro.utils.logger import CliLogger
+
+logger = CliLogger(name=__name__)
+
+
```

Comment:
Do we host the BE ready bin data?

Reply:
Yes. Currently hosting the 10k samples and 336k samples.